sale_procurement_group_by_line/__manifest__.py

```diff
@@ -1,38 +1,23 @@
 # -*- coding: utf-8 -*-
-#
-#
-#    Author: Guewen Baconnier
-#    Copyright 2013-2014 Camptocamp SA
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-#
+# Copyright 2013-2014 Camptocamp SA - Guewen Baconnier
+# © 2016 Eficent Business and IT Consulting Services S.L.
+# © 2016 Serpent Consulting Services Pvt. Ltd.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
 
-{'name': 'Sale Procurement Group by Line',
- 'summary': 'Base module for multiple procurement group by Sale order',
- 'version': '8.0.1.0.0',
- 'author': "Camptocamp,Odoo Community Association (OCA)",
- 'category': 'Warehouse',
- 'license': 'AGPL-3',
- 'complexity': 'expert',
- 'images': [],
- 'website': "http://www.camptocamp.com",
- 'depends': ['sale_stock',
-             ],
- 'demo': [],
- 'data': [],
- 'test': [],
- 'auto_install': False,
- 'installable': False,
- }
+{
+    'name': 'Sale Procurement Group by Line',
+    'summary': 'Base module for multiple procurement group by Sale order',
+    'version': '10.0.1.0.0',
+    "author": "Camptocamp,"
+              "Eficent,"
+              "Serpent,"
+              "Odoo Community Association (OCA)",
+    'category': 'Warehouse',
+    'license': 'AGPL-3',
+    'complexity': 'expert',
```

Comment:
I am wondering what is the use of this `complexity`?

Reply:
This is deprecated